server/webapp/WEB-INF/rails.new/app/assets/javascripts/package_material_check_connection.js

```diff
@@ -15,18 +15,22 @@
  *************************GO-LICENSE-END**********************************/
 
 PackageMaterialCheckConnection = function (url) {
-
     _bind = function (parent_container_selector, check_connection_selector, connection_message_selector) {
         jQuery(parent_container_selector).on('click', check_connection_selector, function (e) {
             var connection_message_element = jQuery(connection_message_selector);
-            connection_message_element.removeClass("error_message").removeClass("ok_message").text('Checking connection...');
+            var formData = jQuery(parent_container_selector + " :input").serialize();
+            formData = formData.replace("_method=PUT", "_method=POST");
```

Comment:
That's not true. the `_method` hack is in rails because of the fact that old browsers did not support some of the verbs like PUT, PATCH, OPTIONS, so everything was submitted as a `POST` with an additional `_method` parameter that rails could use to distinguish between different HTTP verbs.

You should not have to do any `formData.replace` in the javascript, because the relevant form fields should have the correct method on them, and submitting using `POST` should work just fine.